observations/obs_converters/even_sphere/README.rst

```diff
@@ -8,69 +8,113 @@ generate a vertical column of observations.  This could mimic
 a radiosonde observing network, for example.
 
 
-
-This directory contains a MATLAB script that generates
-input for the 'create_obs_sequence' program.  It takes
-a number of vertical levels and a total number of points,
-and generates a roughly evenly distributed set of observations
+This directory contains MATLAB scripts (even_sphere*.m) 
+that generate input for the `create_obs_sequence 
+<../../../assimilation_code/programs/create_obs_sequence/create_obs_sequence.html>`_ program.  
```

Comment:
i don't understand references.  does this need another underscore after it?  others seem to have 2.